mobile/src/main/java/net/activitywatch/android/watcher/UsageStatsWatcher.kt

```diff
@@ -74,6 +77,26 @@ class UsageStatsWatcher constructor(val context: Context) {
         }
     }
 
+    private var alarmMgr: AlarmManager? = null
+    private lateinit var alarmIntent: PendingIntent
+
+    fun setupAlarm() {
+        alarmMgr = context.getSystemService(Context.ALARM_SERVICE) as AlarmManager
+        alarmIntent = Intent(context, AlarmReceiver::class.java).let { intent ->
+            intent.action = "net.activitywatch.android.watcher.LOG_DATA"
+            PendingIntent.getBroadcast(context, 0, intent, 0)
+        }
+
+        val interval = AlarmManager.INTERVAL_HOUR   // Or if testing: AlarmManager.INTERVAL_HOUR / 60
```

Comment:
Would it also be possible to trigger when you open the app in addition to this?
Otherwise you will see 0-60min old data?

Reply:
That's exactly what I do here: https://github.com/ActivityWatch/aw-android/pull/19/files#diff-154058ee34d4cc2612823b93043f8b90R77

Reply:
Aha, nice!